tasks/rh-sign-image/rh-sign-image.yaml

```diff
@@ -93,6 +95,16 @@ spec:
               manifest_digests=$(jq -r '.manifests[].digest' <<< $RAW_OUTPUT)
             fi
 
+            sourceContainerDigest=
+            if [[ $(jq -r ".images.pushSourceContainer" "${DATA_FILE}") == "true" ]] ; then # Default to false
```

Comment:
Is there a risk of the `pushSourceContainer` being not set? If yes I think it makes sense to add the default to false in the `jq` command?

Reply:
other tests do not set it and I believe it works as-is

but I can change it.

what do you recommend?

Reply:
I personally tend to add, but I tested it and it worked fine. So ok for me leaving it as is.

Reply:
I think it is best like this actually so it is the same logic as `push-snapshot`